### PR TITLE
updating sort page to work for single reference

### PR DESCRIPTION
**Pull request by sweng66**
* adding paper into log-in user's mod corpus
* adding paper into log-in user's mod corpus plus directing curators to TET page
* discarding paper from log-in user's mod
* adding "Update Sorting" button on the top (along with the one at the bottom) to discard/add multiple papers at the same time
* also fixed the search code to handle tester properly (for missing PDF icon issue in search result page)

### CODE REVIEW

File: src/actions/searchActions.js [hunk 1]
```diff
@@ -100,7 +100,7 @@ const getSearchParams = (state) => {
     query_fields: state.search.query_fields,
     sort_by_published_date_order: state.search.sortByPublishedDate,
     partial_match: state.search.partialMatch,
-    mod_abbreviation: state.isLogged.oktaMod
+    mod_abbreviation: state.isLogged.testerMod !== 'No' ? state.isLogged.testerMod : state.isLogged.oktaMod
```
Comment (azurebrd) [inline src/actions/searchActions.js:103]: Does this handle the case where there is no okta login and no tester value ?  (I don't know if mod_abbreviation is required for this)

**Review by azurebrd — APPROVED** [review]
I honestly lost track of what's supposed to happen, but the code looks good, thanks @sweng66

File: src/actions/searchActions.js (hunk 1 above)
Comment (sweng66) [reply src/actions/searchActions.js:103]: Good question! In this special case, mod_abbreviation is null, the main PDF identifying code will return nothing so no main PDF icon will show up on the search result page.

Currently, the PDF icon for a paper with a main PDF uploaded by SGD, does not appear on the search results page when viewed by a non-SGD curator, even if they are logged in as an SGD tester. With this update, we will set mod_abbreviation = 'SGD' when a non-SGD curator is logged in as an SGD tester, making sure the main PDF icon is visible for a tester as well.

**Comment by sweng66** [comment]
Thanks @azurebrd for looking over the code.

This fix (adding "Inside & Open", "Inside", "Outside" buttons) is mainly for SGD paper sorting/triaging. One paper at a time.

The "Inside & Open": adding the given paper to mod corpus and redirecting curator to TET page.
"Inside": adding the paper into mod corpus
"Outside": removing the paper out of mod corpus

Sorry this PR is for two stories so it is a bit messing (too lazy to create a new one).

File: src/actions/searchActions.js (hunk 1 above)
Comment (azurebrd) [reply src/actions/searchActions.js:103]: Oh... cool cool, thanks !

**Comment by azurebrd** [comment]
Got, it thanks @sweng66 !

**Pull request merged**